recipes/boost/all/test_package/conanfile.py

```diff
@@ -6,7 +6,7 @@
 
 class DefaultNameConan(ConanFile):
     settings = "os", "compiler", "arch", "build_type"
-    generators = "cmake"
+    generators = ["cmake", "virtualrunenv"]
```

Comment:
Is it possible to build without virtualrunenv? If not, how can you build all other dependencies, because none of them are using it.

Reply:
Yes! It was forgotten while I was testing the packages.